text/0000-metric-handles.md

```diff
@@ -0,0 +1,76 @@
+# Metric Handle API specification
+
+**Status:** `proposed`
+
+Specify the behavior of the Metrics API `Handle` type, for efficient repeated-use of metric instruments.
+
+## Motivation
+
+The specification currently names this concept `TimeSeries`, the object returned by `GetOrCreateTimeseries`, which supports binding a metric to a pre-defined set of labels for repeated use.  This proposal renames these `Handle` and `GetHandle`, respectively, and adds further detail to the API specification for handles.  
```

Comment:
Having `gauge.GetHandle(...)` seems to be okay.
Having a class/type named `Handle` could be confusing.

Reply:
In the Go prototype, there is indeed a `Handle` type but it's for internal use. The types returned by the `GetHandle()` methods in Go PR#100 are like `Float64GaugeHandle`, `Int64CumulativeHandle`. Is this sufficiently clear?